src/processes/final-user-score-process.js

```diff
@@ -11,19 +22,8 @@ const validatePostMessage = (str) => {
 };
 
 const finalUserScoreProcess = async (u, p, pPerf, job) => {
-  const {
-    finalScorePost,
-    previousInteractionScore,
-    applyMultipliesToTotalScore,
-    scoreBasedPostCharacteristics,
-    RecencyScore,
-    ageOfPost,
-    postPerformanceScore,
-    weightPostLongComments,
-  } = require("../utils");
-  const { StatisticPost, PostStatistic } = require("../databases/models");
-  const impression = await StatisticPost.sum("counter");
-  const comment = await PostStatistic.sum("comment_count");
+  const impression = await StatisticPost.sum('counter');
+  const comment = await PostStatistic.sum('comment_count');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are primarily about code organization and readability. The imports have been moved to the top of the file, which is a common practice in JavaScript to make it easier to see all dependencies at a glance. However, this change could potentially impact performance if these modules are not used every time this script runs, as they will be loaded regardless. If these modules are large or expensive to load, it might be better to require them only when needed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->